testing/test_basic_widgets.py

```diff
@@ -337,6 +338,145 @@ class TestForm(View):
                                    u'Widget': u'widget6'}]
 
 
+def test_table_multiple_tbody(browser):
+    class TBodyRow(TableRow):
+        ROW = "./tr[1]"
+        HIDDEN_CONTENT = "./tr[2]/td[1]"
+
+        def __init__(self, parent, index, logger=None):
+            Widget.__init__(self, parent, logger=logger)
+            # We don't need to adjust index by +1 because anytree Node position will
+            # already be '+1' due to presence of 'thead' among the 'tbody' rows
+            self.index = index
+            self.hidden_content = Text(parent=self, locator=self.HIDDEN_CONTENT)
+
+        def __locator__(self):
+            # We don't need to adjust index by +1 because anytree Node position will
+            # already be '+1' due to presence of 'thead' among the 'tbody' rows
+            loc = self.parent.ROW_AT_INDEX.format(self.index)
+            return self.browser.element(loc, parent=self.parent)
```

Comment:
why do the locators return elements from the locators?

Reply:
The Table class behaves the same way. Should it only return the string?

Reply:
@bsquizz based on other docs it seems like a bug to me that `__locator__` should return a string, `__element__` should return the element

i believe this has gotten in by accident(plus it triggers logged warnings)

Reply:
@RonnyPfannschmidt When I wrote the original version, there was no `__element__ ` yet so the table locator was mentioned as an extremely rare example not worth following.